Shamaz/SecondViewController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  SecondViewController.swift
+//  Shamaz
+//
+//  Created by Alexandra Ivanova on 31/03/2020.
+//  Copyright © 2020 Alex Ivanova. All rights reserved.
+//
+
+import UIKit
+
+@available(iOS 13.0, *)
+class SecondViewController: UIViewController {
```

Comment:
Now you have 2 view controllers, you should rename them so the reader can understand what they do.